tests/spread/general/destructive/task.yaml

```diff
@@ -4,7 +4,13 @@ execute: |
   # Check that work dirs *don't* exist
   test ! -d parts -a ! -d stage -a ! -d prime
   
-  run_rockcraft pack --destructive-mode
+  if run_rockcraft pack --destructive-mode; then
```

Comment:
since destructive now works, do you want to update and remove this in a rebase?

Reply:
yes; after this is reviewed I plan to do a cleanup pass to remove leftover comments, this commit, etc